src/mlagility/api/script_api.py

```diff
@@ -92,28 +92,25 @@ def benchmark_script(
             "node and your local machine."
         )
 
-    # Get a specific list of models to process
-    available_scripts = filesystem.get_available_scripts(search_dir)
-
-    # Filter based on the model names provided by the user
-    if benchmark_all:
-        scripts = [os.path.join(search_dir, script) for script in available_scripts]
-    else:
-        user_script_path = os.path.join(search_dir, input_script)
+    # Ignore everything after the '::' symbol, if there is one
+    clean_scripts = [script.split("::")[0] for script in input_scripts]
```

Comment:
Use decode_input_script() so that we aren't hardcoding the magic `split("::")[0]` logic

Reply:
Done